vtrace/qt.py

```diff
@@ -183,7 +183,7 @@ def __init__(self, trace=None, parent=None):
         self.flagviews = {}
 
         reg_groups = trace.archGetRegisterGroups()
-        for name, group in reg_groups:
+        for name, group in reg_groups.items():
```

Comment:
ummm, apparently neither did the VDB RegisterView gui... this one is a big deal, as it breaks RegisterView (so it never is shown).